docs/tracked-extension-resources-are-not-allowed.md

```diff
@@ -0,0 +1,25 @@
+# trackedExtensionResourcesAreNotAllowed
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI (swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Uri-V1-12
+
+## Output Message
+
+{0} is an extension resource and the response schema in {1} operation includes location property. Extension resources of type tracked are not allowed.
+
+## Description
+
+Extension resources cannot be of type tracked.
+
```

Comment:
Extension resources are always considered to be proxy and cannot be of the tracked type.